src/FlipMove.js

```diff
@@ -213,10 +213,14 @@ class FlipMove extends Component<ConvertedProps, FlipMoveState> {
       this.doesChildNeedToBeAnimated,
     );
 
+    // Splitting DOM reads and writes to be peformed in batches
+    const childrenInitialStyles = dynamicChildren.map(child =>
+      this.computeInitialStyles(child),
+    );
     dynamicChildren.forEach((child, n) => {
       this.remainingAnimations += 1;
       this.childrenToAnimate.push(getKey(child));
-      this.animateChild(child, n);
+      this.animateChild(child, n, childrenInitialStyles);
```

Comment:
Nit: I think we can just pass the specific style for this child, rather than the whole object, with `childrenInitialStyles[n]`, since the index `n` will be the same across that first map and the `forEach`.

(Also, we should really change `n` to be `index` to be consistent, but that's on me to tackle later. Sorry for the confusion with variable names!)